sdv/metadata/multi_table.py

```diff
@@ -354,11 +371,30 @@ def detect_table_from_csv(self, table_name, filepath):
         """
         self._validate_table_not_detected(table_name)
         table = SingleTableMetadata()
-        data = table._load_data_from_csv(filepath)
-        table._detect_columns(data)
+        table.detect_from_csv(filepath)
         self.tables[table_name] = table
         self._log_detected_table(table)
 
+    def detect_from_csvs(self, folder_name):
+        """Detect the metadata for all tables in a folder of csv files.
+
+        Args:
+            folder_name (str):
+                Name of the folder to detect the metadata from.
+
+        Raises:
```

Comment:
We never write `raises` in docstrings. I guess for consistency it would make sense not to add it?